src/player/update/update_hurt.c

```diff
@@ -5,22 +5,79 @@
 ** update hurt
 */
 
+#include <SFML/Graphics/View.h>
+#include <SFML/System/Vector2.h>
+#include <stdbool.h>
+#include "ennemies.h"
+#include "my_dico.h"
 #include "my_rpg.h"
 #include "my_json.h"
+#include "math.h"
+#include "maths.h"
 
-static void check_blink_time_end(float *since_start, float dtime, any_t *data,
+static const float time_nulll = 1;
+
+static const float coef = 1000;
+
+const char data_directions[] = "dirrections player hurt";
+
+static int nb_intervals = 20;
+
+static void update_move_player(player_t *player, window_t *win, sfVector2f new)
+{
+    ennemy_t enn = {0};
+    sfVector2f delta_x = {new.x / nb_intervals, new.y / nb_intervals};
+
+    enn = (ennemy_t) {player->state, player->dir, 1, player->obj, 0, 1};
```

Comment:
Magic value ?

Reply:
no, i just transform the player to ennemy to use function with ennemy